### ISSUE TITLE
Location codes are not included in the restriction cache

### ISSUE THREAD

**Issue by jbienkowski** [EIDA/ws-availability#14]
Location codes are not included in the `RESTRICTED_INVENTORY` therefore cannot be used to expand the wildcards here:
https://github.com/EIDA/ws-availability/blob/master/apps/wfcatalog_client.py#L178

![image](https://user-images.githubusercontent.com/8037752/229548212-61d63130-f251-4ab3-afcc-25a1402480d4.png)

**Comment by andres-h** [comment]
Probably related to this issue is that ALL channels with non-empty location code are OPEN, even if parent station and network are restricted. Check our 6A_2020 network as an example.

**Comment by jbienkowski** [comment]
Hi @andres-h,
Can you pull and re-deploy the service? I think latest commits include fix to issue you have mentioned.
Cheers,
Jarek

**Comment by andres-h** [comment]
Sorry about the delay. I just upgraded, but the fix seems incomplete. The following change is required also.

```diff
--- a/apps/wfcatalog_client.py
+++ b/apps/wfcatalog_client.py
@@ -112,7 +112,7 @@ def _apply_restricted_bit(data: list) -> list:
     data = [
         r
         for r in data
-        if f"{r['net']}.{r['sta']}..{r['cha']}"
+        if f"{r['net']}.{r['sta']}.{r['loc']}.{r['cha']}"
         in RESTRICTED_INVENTORY._known_seedIDs
     ]
 
@@ -217,7 +217,7 @@ def _get_restricted_status(segment):
         RESTRICTED_INVENTORY = RestrictionInventory()
 
     r = RESTRICTED_INVENTORY.is_restricted(
-        f"{segment['net']}.{segment['sta']}..{segment['cha']}",
+        f"{segment['net']}.{segment['sta']}.{segment['loc']}.{segment['cha']}",
         segment["ts"].date(),
         segment["te"].date(),
     )
```

**Comment by jbienkowski** [comment]
Thanks, @andres-h, it's fixed with #20.
